doto-frontend/src/components/ModalContent.js

```diff
@@ -38,7 +38,13 @@ const ModalContent = props => {
     const [selectedName, setSelectedName] = useState("TASK - " + new Date());
     const [selectedDescription, setSelectedDescription] = useState("");
     const [selectedDueDate, setSelectedDueDate] = useState(new Date());
-    const [selectedDuration, setSelectedDuration] = useState(0);
+
+    // default duration is 1 hour
+    var initialDuration = new Date();
+    initialDuration.setHours(1);
+    initialDuration.setMinutes(0);
+    const [selectedDuration, setselectedDuration] = React.useState(initialDuration);
```

Comment:
Nitpicking, but why the difference in styling between `setselectedDuration` and `setSelectedLocation`?

Reply:
ah copy and paste error, I'll change that